src/electionguard/types.py

```diff
@@ -1,4 +1,9 @@
+from typing import TypeVar
+
 BALLOT_ID = str
 CONTEST_ID = str
 GUARDIAN_ID = str
 SELECTION_ID = str
+ENCRYPTION_OUTPUT = TypeVar("ENCRYPTION_OUTPUT")
+SECRET_KEY = TypeVar("SECRET_KEY")
+PUBLIC_KEY = TypeVar("PUBLIC_KEY")
```

Comment:
Generic types were used to ensure the Auxiliary keys would be flexible. The hope is to allow flexibility.